packages/accordions/src/styled/accordion/StyledRotateIcon.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { cloneElement, Children } from 'react';
+import styled from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'accordions.rotate_icon';
+
+export interface IStyledRotateIcon {
+  isExpanded?: boolean;
+  isCompact?: boolean;
+}
+
+/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
+export const StyledRotateIcon = styled(({ children, isRotated, isHovered, isCompact, ...props }) =>
+  cloneElement(Children.only(children), props)
+).attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledRotateIcon>`
+  transform: ${props => props.isRotated && `rotate(${props.theme.rtl ? '-' : '+'}180deg)`};
+  transition: transform 0.25s ease-in-out;
+  padding: ${props =>
+    props.isCompact
+      ? `${props.theme.space.base * 1.5}px ${props.theme.space.base * 3}px`
```

Comment:
Also base-4 sizing. 4px or 8px

Reply:
>i remember this conversation. did we end up changing the isCompact padding on the rest of our components to base-4? currently i still see them at 6px and 12px for vertical and horizontal padding respectively.

to clarify, i'm completely onboard on updating this to base-4, i just want some clarification on the whole system's perspective on this spacing so the accordions can match.